parquet/read.go

```diff
@@ -188,12 +189,10 @@ func reverseTransformInet(col *array.String) arrow.Array {
 func reverseTransformUUID(col *array.String) arrow.Array {
 	bldr := types.NewUUIDBuilder(array.NewExtensionBuilder(memory.DefaultAllocator, types.ExtensionTypes.UUID))
 	for i := 0; i < col.Len(); i++ {
-		if col.IsNull(i) {
+		if !col.IsValid(i) || col.Value(i) == "" {
```

Comment:
This made it work in some sense but I've no idea why this is needed. it Should be invalid.

Reply:
yes I think we have an issue persisting the null bitmap while writing.